code/renderers/vue3/template/stories/ReactiveArgs.stories.js

```diff
@@ -42,3 +42,86 @@ export const ReactiveTest = {
     await expect(reactiveButton).toHaveTextContent('updated 2');
   },
 };
+
+export const ReactiveHtmlWrapper = {
+  args: { label: 'Wrapped Button' },
+
+  decorators: [
+    () => ({
+      template: `
+        <div style="border: 5px solid red;">
+          <story/>
+        </div>
+        `,
+    }),
+  ],
+  play: async ({ canvasElement, id }) => {
+    const channel = globalThis.__STORYBOOK_ADDONS_CHANNEL__;
+    const canvas = within(canvasElement);
+
+    await channel.emit(RESET_STORY_ARGS, { storyId: id });
+    await new Promise((resolve) => {
+      channel.once(STORY_ARGS_UPDATED, resolve);
+    });
+    const reactiveButton = await canvas.getByRole('button');
+    await expect(reactiveButton).toHaveTextContent('Wrapped Button 0');
+
+    await userEvent.click(reactiveButton); // click to update the label to increment the count + 1
+    await channel.emit(UPDATE_STORY_ARGS, {
+      storyId: id,
+      updatedArgs: { label: 'updated Wrapped Button' },
+    });
+    await new Promise((resolve) => {
+      channel.once(STORY_ARGS_UPDATED, resolve);
+    });
+    await expect(canvas.getByRole('button')).toHaveTextContent('updated Wrapped Button 1');
+
+    await userEvent.click(reactiveButton); // click to update the label to increment the count + 1
+    await expect(reactiveButton).toHaveTextContent('updated Wrapped Button 2');
+  },
+};
+
+// to test that Simple html Decorators in CSF2 format are applied correctly in reactive mode
+export const ReactiveCSF2Wrapper = (args, { argTypes }) => ({
```

Comment:
these 2 Stories to show the issue on the next branch, Stories are not reactive they decorated, whether CSF3 or CSF2